src/components/atoms/BoxToken/BoxToken.tsx

```diff
@@ -107,11 +111,11 @@ export const BoxToken = (props: BoxTokenProps): JSX.Element => (
         </tr>
       </thead>
       <tbody>
-        {props.apiTokens
+        {apiTokens
```

Comment:
Just curious - couldn't this have been fixed by duck typing? Like just changing this line to `apiTokens && apiTokens.length && apiTokens.map...`

Reply:
yes, it works. But I prefer to avoid unnecessary logic.

What do you prefer?

Reply:
LGTM